README.md

```diff
@@ -47,10 +47,10 @@ jobs:
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so that typilus can access it.
     - uses: actions/checkout@v2
-    - uses: typilus/typilus-action@master
+    - uses: typilus/typilus-action@v0.9
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        MODEL_PATH: path/to/model.pkl.gz   # Optional: provide the path of a custom model instead of the pre-trained model.
+        # MODEL_PATH: path/to/model.pkl.gz   # Optional: provide the path of a custom model instead of the pre-trained model.
```

Comment:
the repos using this action do disable this line. I think it's easier for a beginner to start with a simple code